### PR TITLE
Standardize edxapp env setup

### PR DESCRIPTION
**Pull request by maxrothman**
Bring edxapp in line with other IDAs for how it sets its environment
variables. This includes adding shell scripts in between supervisor conf
files and gunicorn files.

@edx/devops

### CODE REVIEW

**Comment by e0d** [comment]
Seeing this:

```
ERROR: Syntax Error while loading YAML script, /edx/app/edx_ansible/edx_ansible/docker/plays/roles/edxapp/defaults/main.yml
Note: The error may actually appear before this position: line 633, column 7

edxapp_lms_environment:
  <<: *exdapp_common_environment
      ^
The command '/bin/sh -c sudo ansible-playbook edxapp.yml -c local -e "EDXAPP_PYTHON_SANDBOX=false" -t 'install:base'' returned a non-zero code: 1
```

**Comment by maxrothman** [comment]
Whoops, that was a typo.

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -622,6 +623,27 @@ edxapp_environment:
   # yaml based configs is complete
   CONFIG_ROOT: "{{ edxapp_app_dir }}"
 
+  #NB: other apps (e.g. ecommerce) set these conditionally in *.sh.j2.
+  #In following the pattern set by how we handle settings files, I prefer this approach.
+  #Would like to discuss which we prefer before merging.
```
Comment (nedbat) [inline playbooks/roles/edxapp/defaults/main.yml:628]: Nit: The comment would be more readable with a space after the hash.

**Comment by maxrothman** [comment]
Closing in favor of #2512

**Pull request closed without merging**